src/core/utils/default_app_categories.ts

```diff
@@ -73,4 +73,25 @@ export const DEFAULT_APP_CATEGORIES: Record<string, AppCategory> = Object.freeze
     order: 5000,
     euiIconType: 'managementApp',
   },
+  investigate: {
```

Comment:
Are these new categories added by intentionally? Seems not related the new interface.

Reply:
Yes, is is added since these categories are default categories for core applications.